command/attach_artifacts.go

```diff
@@ -15,8 +15,12 @@ import (
 )
 
 type attachArtifacts struct {
+	// Files is a list of files, using gitignore syntax.
 	Files []string `mapstructure:"files" plugin:"expand"`
 
+	// Prefix is an optional directory prefix to start file globbing in, relative to Evergreen's working directory.
+	Prefix string `mapstructure:"working_dir" plugin:"expand"`
```

Comment:
I don't know that this is consistent with the way that other working_dir settings work.